config/rbac/role.yaml

```diff
@@ -167,8 +167,11 @@ rules:
   resources:
   - inferenceservices
   verbs:
+  - create
```

Comment:
Why do we need this 2 additional verbs?

Reply:
Good question, that is because the new reconciliation will create/delete/update `inferenceservices` objects based on the Model Registry content